src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/SymbolEquivalenceComparer.cs

```diff
@@ -238,6 +238,12 @@ private static bool IsPartialMethodDefinitionPart(IPropertySymbol symbol)
     private static bool IsPartialMethodImplementationPart(IPropertySymbol symbol)
         => symbol.PartialDefinitionPart != null;
 
+    private static bool IsPartialMethodDefinitionPart(IEventSymbol symbol)
```

Comment:
It might be good to name these according to the symbol kinds they actually deal with, i.e. `IsPartialEventDefinitionPart`. Not critical, though.